### ISSUE TITLE
Calendar (year selector)[BUG]

### ISSUE THREAD

**Issue by RasmusTraeholt** [kirbydesign/designsystem#2052]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
Not possible to scroll the last element of the year selector dropdown into view (which makes it impossible to select the last element in the list).

## Describe how to reproduce the bug
Place a calendar in any context with yearNavigatorOptions that holds more than 8-10 years (the list needs to be longer than the height of the calendar component).

## Which Kirby version was used?
5.0.18

## What was the expected behavior?
To able to select the last element in the year navigator dropdown

## Add any screenshots
![image-2022-02-08-16-44-37-617](https://user-images.githubusercontent.com/23026215/154490152-77165fcd-53d1-4d45-81a0-c0fb59bd079e.png)

## Please complete the following information:
- OS: seen on iOS and android
- Browser: seen on firefox and chrome webbrowser
- Browser version: latest

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/master/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/master/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by MadsBuchmann** [comment]
Duplicate of https://github.com/kirbydesign/designsystem/issues/1411. Closing. 
@alxzak - we should prioritize 1411 instead as it refers to the same bug.
